src/client/apps/edit/components/admin/components/article/article_authors.tsx

```diff
@@ -45,7 +39,7 @@ export class ArticleAuthors extends Component {
         .query({ query: AuthorsQuery(idsToFetch) })
         .end((err, res) => {
           if (err) {
-            console.error(err)
+            new Error(err)
```

Comment:
I wonder if we ever want to raise an exception. We also may want to disable the TSLint inspection for `console.log`. @damassi Any thoughts?

Reply:
In clientside code, unless it's totally necessary to stop execution and break the UX we should avoid raising exceptions like this. `console.error` (and perhaps a log to sentry) is more ideal.

Reply:
For a simple Sentry / logger example, see https://github.com/artsy/prediction/blob/master/src/utils/logger.js

Reply:
We have an error boundary to catch client side errors and display them nicely, but yes, `console.error` is not approved by tslint

Reply:
Ah cool, if things are being routed to the ErrorBoundary that seems ideal. 

If you did want to turn it off could do 

```
"no-console": [true, "log"]
```
That way it only guards against log, but allows "warn" and "error". 

We don't yet have a unified linting setup across artsy so in the meantime feel free to adjust based on app preferences.